content/markdown/03-plotting/03-01-intro-to-ggplot.rmd

```diff
@@ -0,0 +1,57 @@
+---
+output:
+  pdf_document: default
+  html_document: default
+---
+# Introduction to `ggplot2`
+**Author:** Cole Brookson
+**Date:** 20 July 2022
+
+```{r, echo=FALSE}
+options(tidyverse.quiet = TRUE)
+```
+
+One way to plot in R, is to use the `ggplot2` package in the Tidyverse. "gg" stands for the "Grammar of Graphics" and the package makes for a convenient and consistent way to make almost any plot you want. Typically it's easiest to plot data in `ggplot2` using a dataframe, which means we usually don't need to make too major of changes to our data if we decide we want to change how we're plotting something in particular. 
```

Comment:
I would edit last sentence to read: "which means we usually don't need to make major changes to our data..."